classes/Lesson.php

```diff
@@ -563,8 +564,14 @@ public function tutor_lesson_completed_after( $content_id ) {
 	 */
 	public function reply_lesson_comment() {
 		tutor_utils()->checking_nonce();
+		$comment = Input::post( 'comment', '', Input::TYPE_KSES_POST );
+		if ( 0 === strlen( $comment ) ) {
+			wp_send_json_error();
```

Comment:
We  can send a message with `wp_send_json_error` to let the user know the exact reason for failure